.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I believe we still want the config to live in `package.json`. This was done intentionally to reduce errors when the project was copied. See #405

Reply:
Ok no prob)